test/SwiftSyntax/AbsolutePosition.swift

```diff
@@ -101,4 +101,31 @@ PositionTests.test("Recursion") {
   })
 }
 
+PositionTests.test("Trivias") {
+  expectDoesNotThrow({
+    var l = [CodeBlockItemSyntax]()
+    let leading = Trivia.newlines(1).appending(TriviaPiece.backticks(1))
```

Comment:
This could be cleaner as:

```swift
let leading = Trivia(pieces: [
  .newlines(1),
  .backticks(1),
  .docLineComment("/// some comment")
])
```